js/events.js

```diff
@@ -1,9 +1,25 @@
-import { createEventElement, createMultiDayEventElement, getFirstDayOfTheWeek, clearEvents, differenceBetweenTwoDatesInDays} from './utils.js';
+import { createEventElement, createMultiDayEventElement, getFirstDayOfTheWeek, clearEvents, differenceBetweenTwoDatesInDays, dateIsInRange} from './utils.js';
 import { getEventsFromLocalStorage } from './services.js';
 
 const HOUR_IN_MINUTES = 60;
+const doesEventOverlapWithOtherEvents = (eventCurrent) => {
+  const eventStartDateTime = new Date(eventCurrent.startDateTime);
+  const eventEndDateTime = new Date(eventCurrent.endDateTime);
+  const events = getEventsFromLocalStorage();
```

Comment:
we do not have saved events state atm so my comment does not really apply : )
but just for future - if getting events was a network call we would like to be cautious if and when we make network calls. we might get to this when rewriting to react, not sure